### ISSUE TITLE
actionmenu: it's possible to pop nested menu but have parent item hover style still active

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#448]
How?  Who knows!

https://pluralsight.slack.com/archives/C70HPSJPP/p1547673745030400

If it's a mouse interaction first and then keyboard later, I've observed that.

**Comment by jaketrent** [comment]
Hover a item with no nested menu, gets hover style.
Hover a item with nested menu, gets hover style, but the no nested menu item remains hover-styled.
No nested menu item retains hover style until a different no nested menu item is hovered.

**Comment by jaketrent** [comment]
We spent a fair bit to find the issue here.... turns out it was working, but because the docs pass `shouldFocusOnMount=false`, the behavior is off, where hover/focus style isn't being transferred as expected.

So, we decided that for now:
1. No fix/change to the src
2. Docs remain as-is in the multiple examples on the page already
3. Add a new in-app example with a button trigger that will allow us to focus on mount and then see a more true-to-life example of how it should work. 

cc @michaelkushlan
